utilities.py

```diff
@@ -751,6 +910,32 @@ def build_plots(
         include_plotlyjs = 'cdn',
     )
 
+    # To creat cal code distributions scatter
+
+    # Get list of all board ids
+    board_ids = sorted(df["data_board_id"].unique())
+    range_cal = [0, 1024]
+
+    # Copied pivot_df from build_time_plots
+    # Create the pivot table with a column for each board
+    pivot_df = df.pivot(
+        index = 'event',
+        columns = 'data_board_id',
+        values = list(set(df.columns) - {'data_board_id', 'event'}),
+    )
+    # Rename columns so they are no longer hierarchical
+    pivot_df.columns = ["{}_{}".format(x, y) for x, y in pivot_df.columns]
+
+    make_calcode_correlation_plots(
+        data_df=pivot_df,
+        base_path=base_path,
+        run_name=run_name,
+        board_ids=board_ids,
+        range_cal=range_cal,
```

Comment:
This line is not needed since line 917 is the default value. Remove this line and line 917.

```suggestion
```

Reply:
I removed the range_cal line